.github/workflows/main.yml

```diff
@@ -100,13 +100,18 @@ jobs:
           - '13.4.1' # Swift 5.6.1
     steps:
       - uses: actions/checkout@v2
+      - uses: apple-actions/import-codesign-certs@v2
+        with:
+          p12-file-base64: ${{ secrets.SIGNING_CERTIFICATE_BASE_64 }}
+          p12-password: ${{ secrets.SIGNING_CERTIFICATE_PASSWORD }}
```

Comment:
I followed these steps to create a self-signing certificate and installed it as a secret in this repository:

 - Create code signing certificate in Keychain Assistant:
    - First you have to create a custom Certificate Authority in order to issue a certificate that can be used for code signing. Follow these instructions: https://www.simplified.guide/macos/keychain-ca-code-signing-create
    - Then you can create a certificate for code signing. Follow these instructions: https://www.simplified.guide/macos/keychain-cert-code-signing-create
- Export .p12 file for both the code signing certificate and the root certificate authority, protected with a password.
- Add certificate password as GitHub actions secret (`SIGNING_CERTIFICATE_PASSWORD`)
- Convert certificate file to base64, e.g. using `base64 -i "Lottie iOS Code Signing.p12"`.
- Added base 64 as secret to GitHub repo, e.g. using `SIGNING_CERTIFICATE_BASE_64` key